examples/05_stable_diffusion/scripts/compile.py

```diff
@@ -57,6 +54,10 @@ def compile_diffusers(
         torch_dtype=torch.float16,
     ).to("cuda")
 
+    assert (
+        height % 64 == 0 and width % 64 == 0
+    ), "Height and Width must be multiples of 64, otherwise, the compilation process will fail."
+
```

Comment:
to add some context into the error message,
```suggestion
f"got {height=} {width=}"
```